### ISSUE TITLE
Refactor Tasks.Feed validation in light of new build manifest xml

### ISSUE THREAD

**Issue by JohnTortugo** [dotnet/arcade#2201]
Refactor [Tasks.Feed input validation](https://github.com/dotnet/arcade/blob/a0d951e12a53401c3da812b581e39feb4a5bb4ab/src/Microsoft.DotNet.Build.Tasks.Feed/src/BuildManifestUtil.cs#L102) considering the added/removed fields of the new publishing using pipelines mechanism.

**Comment by riarenas** [comment]
Refactor how? What's the problem, and what's the expected outcome?

**Comment by JohnTortugo** [comment]
With the new infra we added/deprecated some fields in the build manifest. The current validation mostly validate old fields (looks like just one by the way). We need to improve it to validate the new fields and check if we should be validating the the old ones as well or not.

A double check is needed to see if there is any task in Tasks.Feed that generate manifests in the old format. If there is we need to see if these tasks are really needed now that we moved to stages and/or if we can move to the new manifest format.
